src/cli/load.js

```diff
@@ -113,6 +128,23 @@ const load: Command = async (args, std) => {
     }
   }
 
+  if (organization != null) {
+    if (repoIds.length > 0) {
+      die(std, "'--organization' cannot be used in conjunction with repoIds");
+    }
+    if (explicitOutput == null) {
+      explicitOutput = makeRepoId(organization, "combined");
```

Comment:
\*sigh\*, yeah. I guess we don’t really have a better mechanism than
faking a repository ID for now.